samples/Commerce/Commerce.Shared/Views/CartPage.xaml

```diff
@@ -47,7 +47,9 @@
 											   Style="{StaticResource MaterialSubtitle2}" />
 								</utu:AutoLayout>
 								<ListView ItemsSource="{Binding Items}"
-										  uen:Navigation.Request="ProductDetails"
+										  uen:Navigation.Request=""
```

Comment:
What does it mean to navigate to an empty string ?

Reply:
It means that navigation should be based on the data type of the item selected from the list. In this case, it's a CartItem - so this type is used to look up the route from the table of RouteMaps